00-open-a-shapefile.Rmd

```diff
@@ -160,9 +163,9 @@ aoiBoundary
 Here we see that R interprets the data as a SpatialPolygonsDataFrame data class
 and information about the coordinate reference system including that the 
 coordinates  are UTMs from zone 18.  This helps us interpret the previous line 
-giving the extent of the file.  We will discuss CRS and extent a little later.
-Finally we see there is one variable named "id" that appears to consists of only 
-ones (min and max value both 1).
+giving the extent of the file.  We will discuss metadata and attributes in more
+detail a bit later. Finally we see there is one variable named "id" that appears
+to consists of only ones (min and max value both 1).
```

Comment:
Ok how about this - let's open an issue that points to this file and this area of the file  -- i think the bigger issue is that this particular shapefile is boring and only has one meaningless attribute. 

i can either 1.come up with a better file that has more than one point (probably a good idea)
OR 
2. add attributes to this particular file (also probably a good thing to do. Go ahead and open an issue AND let's revisit this while accepting the pull request as is for now. thank you!